src/components/ListboxGrid/distribute-resources.ts

```diff
@@ -13,4 +13,48 @@ const distributeResources = (resources: IListboxResource[], columnCount: number)
   return { firstColumn, restOfColumns };
 };
 
-export default distributeResources;
+const removeOverflowingItems = (height: number, foldedItemHeight: number, count: number, rescources: IListboxResource[]) => {
+  const sliceIndex = (height / foldedItemHeight) - count - 2;
```

Comment:
Not super clear to me how this calculation works. If a comment can make it easier to understand then I will support that.